CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# Release 1.8.0 12-09-2017
+
+- add a test case for OPTIONS. (#261)
```

Comment:
we only list of bug fix and new features in the CHANGE log.  not internal changes.

Reply:
removed